src/pylexibank/report.py

```diff
@@ -0,0 +1,193 @@
+import collections
+
+from clldutils.misc import nfilter
+
+from pylexibank.util import get_badge
+
+
+def build_status_badge(dataset):
+    if dataset.dir.joinpath('.travis.yml').exists() and dataset.repo and dataset.repo.github_repo:
+        return "[![Build Status](https://travis-ci.org/{0}.svg?branch=master)]" \
+               "(https://travis-ci.org/{0})".format(dataset.repo.github_repo)
+    return ''
+
+
+def report(dataset, tr_analysis=None, glottolog=None, log=None):
+    #
+    # FIXME: write only summary into README.md
+    # in case of multiple cldf datasets:
+    # - separate lexemes.md and transcriptions.md
+    #
+    lines = []
+
+    # add NOTES.md
+    if dataset.dir.joinpath('NOTES.md').exists():
+        lines.append('## Notes\n')
+        lines.append(dataset.dir.joinpath('NOTES.md').read_text() + '\n\n')
+
+    badges = nfilter([build_status_badge(dataset)])
+
+    for cldf_spec in dataset.cldf_specs_dict.values():
+        lines.extend(cldf_report(cldf_spec, tr_analysis, badges, log, glottolog))
+        break
+    return '\n'.join(lines)
+
+
+def cldf_report(cldf_spec, tr_analysis, badges, log, glottolog):
```

Comment:
Cool, much cleaner, than the mess we had before. Can we expose this as a CLI command too?

Reply:
We could - but at some point we decided to roll it into `_cmd_makecldf`, presumably because this will re-use initialized Glottolog and Concepticon?

Reply:
Hmm ok. Is it useful to be able to run the report separately too?